app/controllers/site_controller.rb

```diff
@@ -565,7 +565,7 @@ def create_user_annotations
         @options = load_cluster_group_options
         @notice = nil
         @alert = 'The following errors prevented the annotation from being saved: ' + @user_annotation.errors.full_messages.join(',')
-        logger.error "#{Time.now}: Creating user annotation of params: #{user_annotation_params}, unable to save user annotation #{e.message}"
```

Comment:
rather than just calling @user_annotation.errors, we should call @user_annotation.errors.full_messages.join(', ') so that it prints everything to the log (otherwise we just see an object and not the messages)